### PR TITLE
YLP-498 Pending invitations

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/pages/hcp/team-card.tsx [hunk 1]
```diff
@@ -168,16 +168,18 @@ function TeamCard(props: TeamCardProps): JSX.Element {
...
+    }
   };
   const handleClickAddMember = async (): Promise<void> => {
     setButtonsDisabled(true);
     await onShowAddMemberDialog(team);
     setButtonsDisabled(false);
   };
 
-  const { id } = team;
+  const id = team.id;
```
Comment (leovalette) [inline packages/yourloops/pages/hcp/team-card.tsx:182]: Why ?

**Review by leovalette — COMMENTED** [review]
I would like to avoid the use of classes as much as possible. As we move towards the use of functional components, we can keep this spirit in all the code and not only for components.

Here, I think classes could just be replaced by interfaces

File: packages/yourloops/pages/hcp/team-card.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/team-card.tsx:182]: it's a getter, not a basic property.

**Comment by yvon-dblg** [comment]
> 
> 
> I would like to avoid the use of classes as much as possible. As we move towards the use of functional components, we can keep this spirit in all the code and not only for components.
> 
> Here, I think classes could just be replaced by interfaces

For me, interface + object + helper methods = classes.
So I don't really understand your point.

Also, theses are not functional components.

**Comment by leovalette** [comment]
> > I would like to avoid the use of classes as much as possible. As we move towards the use of functional components, we can keep this spirit in all the code and not only for components.
> > Here, I think classes could just be replaced by interfaces
> 
> For me, interface + object + helper methods = classes.
> So I don't really understand your point.
> 
> Also, theses are not functional components.

Why not make functional components instead of class components ? React-team is pushing for it.

**Comment by yvon-dblg** [comment]
Replaced by https://github.com/mdblp/blip/pull/402
Another one will be opened for the YLP-498.

**Pull request closed without merging**